app/server/sonicpi/lib/sonicpi/synthinfo.rb

```diff
@@ -1555,6 +1555,34 @@ def arg_defaults
         :vibrato_onset => 0.1,
       }
     end
+
+    def specific_arg_info
+      {
+        :vibrato_rate => {
+          :doc => "Number of wobbles per second. For realism this should be between 6 and 8, maybe even faster for really high notes.",
+          :validations => [v_greater_than_oet(:vibrato_depth, 0.0), v_less_than_oet(:vibrato_depth, 20.0)],
```

Comment:
@xavriley @samaaron shouln't *:vibrato:depth* be replaced with *:vibrato_rate* in this line?

```ruby
:validations => [v_greater_than_oet(:vibrato_rate, 0.0), v_less_than_oet(:vibrato_rate, 20.0)],
```